lib/Reader/readerir.cpp

```diff
@@ -7462,11 +7462,15 @@ void GenIR::addPHIOperand(PHINode *Phi, Value *NewOperand,
     LLVMBuilder->restoreIP(SavedInsertPoint);
   }
 
-  int BlockIndex = Phi->getBasicBlockIndex(NewBlock);
-  if (BlockIndex >= 0)
-    Phi->setIncomingValue(BlockIndex, NewOperand);
-  else
-    Phi->addIncoming(NewOperand, NewBlock);
+  bool FoundBlockOperand = false;
```

Comment:
Does this generate a warning in release builds?  Might need to wrap this in #ifndef NDEBUG.